react-native-gutenberg-bridge/index.js

```diff
@@ -1,11 +1,25 @@
 /** @format */
 
-import { NativeModules, NativeEventEmitter } from 'react-native';
+import { NativeModules, NativeEventEmitter, Platform } from 'react-native';
 
 const { RNReactNativeGutenbergBridge } = NativeModules;
+const isIOS = Platform.OS === 'ios';
 
 const gutenbergBridgeEvents = new NativeEventEmitter( RNReactNativeGutenbergBridge );
 
+// Send messages
+
+export function sendNativeEditorDidLayout() {
+	// For now, this is only needed on iOS to solve layout issues with the toolbar.
+	// If this become necessary on Android in the future, we can try to build a registration API from Native
+	// to register messages it wants to receive, similar to the Native -> JS messages listener system.
+	if ( isIOS ) {
+		RNReactNativeGutenbergBridge.editorDidLayout();
+	}
```

Comment:
Avoid breaking Android example app.
This is the approach we decided to take with @hypest